src/lib/Scenarios.ts

```diff
@@ -1,7 +1,26 @@
-import { WorldSetup } from "./SimulatorState";
+import { RandomEvent } from './RandomEvents';
+import { VictoryCondition } from './VictoryConditon';
+/**
+ * Represents the initial state of the game world.
+ */
+export interface Scenario {
+    totalPopulation: number;
+    initialNumInfected: number;
+    importedCasesPerDay: number;
+    r0: number;
+    hospitalCapacity: number;
+    gdpPerDay: number;
+    power: number;
+    distr_family: string;
+    dynamics: string;
+    mortality: number; // A number between 0 and 1 representing the mortality
+    time_lumping: boolean;
+    randomEvents: RandomEvent[];
```

Comment:
Added the list of available random events and victory conditions for the scenario